linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_certified/rule.yml

```diff
@@ -10,6 +10,8 @@ description: |-
     Linux vendor, Red Hat, Inc. is responsible for providing security patches as well
     as meeting and maintaining goverment certifications and standards.
 
+    There is no remediation besides switching to a different operating system.
```

Comment:
@mpreisler can you make this a warning to highlight this a bit more?

Reply:
@mpreisler reminder ;)

Reply:
DONE. Much <3